tests/test_schema.py

```diff
@@ -916,24 +916,22 @@ class Model(BaseModel):
     assert model_schema == {
         'title': 'Model',
         'type': 'object',
+        'properties': {'d': {'$ref': '#/definitions/Baz'}},
```

Comment:
isn't this just moved around? If so, could you move it back?

Reply:
Yep. Copy-paste conflict. Fixed.